docs/04-operation-guides/operations/smsh-01-istio-enable-sidecar-injection.md

```diff
@@ -12,7 +12,9 @@ You can enable sidecar proxy injection for either an entire Namespace or a singl
    kubectl label namespace {YOUR_NAMESPACE} istio-injection=enabled
    ```
 
-* To enable sidecar proxy injection for a Deployment, add this annotation to the Deployment configuration file: `sidecar.istio.io/inject: "true"`
+* To enable sidecar proxy injection for a Deployment, you can add this as either label or annotation to the Deployment configuration file: `sidecar.istio.io/inject: "true"`
+
+Please note that the Namespace label will take precedence before the Pod label or annotations.
```

Comment:
```suggestion
Note that the Namespace label takes precedence over the Pod label or annotation.
```
- Our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#voice-and-tone) say to avoid unnecessary words, such as `please`, that don't add anything. 
- Something takes precedence **over** something else. See the [Cambridge Dictionary](https://dictionary.cambridge.org/dictionary/english/precedence) for reference. 
- Did you put `annotations` in plural here on purpose?